app/src/main/java/uk/ac/ebi/atlas/monitoring/HealthCheckController.java

```diff
@@ -15,8 +15,8 @@
     public HealthCheckController(HealthCheckService healthCheckService) {
         healthChecker = new HealthChecker(
                 healthCheckService,
-                ImmutableSet.of("atlas-bioentities"),
-                ImmutableSet.of("bulk-analytics"));
+                ImmutableSet.of(), // still needs to receive a collection of 'solr collections'
+                ImmutableSet.of("bulk-analytics", "atlas-bioentities")); // collectio of 'solr aliases'
```

Comment:
Very minor one - in the code comments typo mistake 'collectio'. Please add `n `.

Reply:
I have corrected this, thanks.